README.md

```diff
@@ -85,32 +84,34 @@ We tested `Herculens` with the versions mentioned in parenthesis. It does not me
 - `emcee` (3.0.2), for Ensemble MCMC sampling
-- `lenstronomy` (1.6.0), for particle swarm optimization
+- `lenstronomy` (1.9.3), for particle swarm optimization
 - `palettable` (3.3.0), for nicer colormaps for plots
 
-To run the notebooks, `jupyter` is (of course) also necessary, along with `matplotlib` for plotting.
+To run the notebooks, `jupyter` and `matplotlib` are also necessary.
 
 
 
-## Open-source development
-
-`Herculens` is currently being actively developed, and several science projects will bring many improvements and new features along the way. Moreover, you can expect very soon better documentation and unit testing to increase the robustness of the code.
-
-Anyone that would like to contribute to `Herculens` is highly encouraged to do so. Please do not hesitate to contact us, open issues on github, fork the repository, and suggest improvements!
-
-
-
-
-
-#### A note regarding `lenstronomy`
-
-You may quickly realize that the syntax and naming conventions of `Herculens` share many similarities with the general purpose lensing software package [`lenstronomy`](https://github.com/sibirrer/lenstronomy) ([Birrer et al. 2021](https://joss.theoj.org/papers/10.21105/joss.03283), and references therein). The reason is simple: because it is overall easy to use and open-source, `lenstronomy` was used a as baseline for the early development phases of `Herculens`. Since then, there have been many modifications throughout the code, primarily to add powerful features from `JAX`, then to implement new modeling techniques that were unachievable beforehand. While these features could not be implemented in `lenstronomy` without significantly affecting the original spirit of the code, the proximity between `Herculens`and `lenstronomy` makes it really easy to switch from one to the other.
+## Attribution
 
+### Citation
 
+If you make use of `herculens`, please cite [Galan et al. 2022](https://arxiv.org/abs/2207.05763). This work presents the global architecture of the tool, and applies it to reconstruct a pixelated lens potential with wavelets.
+```
+@article{Galan:2022:herculens,
+    author        = "Galan, Aymeric and Vernardos, Georgios and Peel, Austin and Courbin, Fr\'ed\'eric and Starck, Jean-Luc",
+    title         = "{Using wavelets to capture deviations from smoothness in galaxy-scale strong lenses}",
+    eprint        = "2207.05763",
+    archivePrefix = "arXiv",
+    primaryClass  = "astro-ph.CO",
+    year          = "2022"
+}
+```
 
 
-## Parallelization and GPU support
+### Special mention regarding Lenstronomy
+Part of the `herculens` code originates from the open-source lens modeling software package [`lenstronomy`](https://github.com/sibirrer/lenstronomy), described in [Birrer et al. 2021](https://joss.theoj.org/papers/10.21105/joss.03283) (and references therein). In every source file, proper credits are given to the specific developers and contributors to both the original `lenstronomy` (up to version 1.9.3) and `herculens`.
```

Comment:
In your publication you referenced as follow: "The code structure and part of the modeling routines of Herculens are based on the open-source modeling software package Lenstronomy (Birrer & Amara 2018; Birrer et al. 2021)" which I find ok (you can be more detailed what part here, but no need since you added it in all files)

Reply:
I am not sure to understand. The structure of any lensing code should resemble to the figure you mentioned. And after the first two rows, Herculens is in fact entirely disconnected from Lenstronomy. The equivalent figure in [our paper](https://arxiv.org/pdf/2207.05763.pdf) is very different as well (even though it is a fairly standard flow-chart).

Reply:
ok, I just thought instead of 'Part of the herculens code originates from...' that 'The code structure and part of the modeling routines originate from...'